README.md

```diff
@@ -33,21 +25,19 @@ You can import `vega-tooltip` directly from [`jsDelivr`](https://www.jsdelivr.co
 <script src="https://cdn.jsdelivr.net/npm/vega-lite@2"></script>
 
 <script src="https://cdn.jsdelivr.net/npm/vega-tooltip@[VERSION]"></script>
-<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/vega-tooltip@[VERSION]/build/vega-tooltip.min.css">
 ```
 
 ## APIs
 
-For Vega-Lite: [`vegaTooltip.vegaLite(vgView, vlSpec[, options])`](docs/APIs.md#vltooltip)
-
-For Vega: [`vegaTooltip.vega(vgView[, options])`](docs/APIs.md#vgtooltip)
+[`vegaTooltip(view[, options])`](docs/APIs.md#tooltip)
 
 ## Tutorials
 1. [Creating Your Tooltip](docs/creating_your_tooltip.md)
 2. [Customizing Your Tooltip](docs/customizing_your_tooltip.md)
 3. [Examples and Code on `bl.ocks.org`](https://bl.ocks.org/sirahd)
 
 ## Run Instructions
-1. In the project folder `vega-tooltip`, type command `npm install` to install dependencies.
-2. Then, type `npm start`. This will build the library and start a web server.
+
+1. In the project folder `vega-tooltip`, type command `yarn` to install dependencies.
```

Comment:
here just `yarn` is fine because this is about the development of tooltips itself, so no need for both.

Reply:
👍